superset/views/core.py

```diff
@@ -1707,15 +1709,16 @@ def warm_up_cache(self):
                     form_data=form_data,
                     force=True,
                 )
-                obj.get_json()
+                payload = obj.get_payload()
+                error = payload["error"]
+                status = payload["status"]
             except Exception as ex:
-                logger.exception("Failed to warm up cache")
-                return json_error_response(utils.error_msg_from_exception(ex))
-        return json_success(
-            json.dumps(
-                [{"slice_id": slc.id, "slice_name": slc.slice_name} for slc in slices]
-            )
-        )
+                error = utils.error_msg_from_exception(ex)
+                status = None
+
+            result.append({"slice_id": slc.id, "error": error, "status": status})
```

Comment:
this is the error and the status coming from the `get_viz` function, so perhaps this should be `viz_error` and `viz_status`? That seems clearer than payload_error/status since payload doesn't have a lot of meaning in the scope of a `warm_up_cache` function (it makes me ask "what payload?")